src/components/table/table.jsx

```diff
@@ -33,12 +32,12 @@ Table.propTypes = {
   className: PropTypes.string,
   children: PropTypes.node,
   size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg']),
-  minWidth: PropTypes.number,
+  minWidth: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   style: PropTypes.object,
 };
 
 Table.defaultProps = {
-  minWidth: 700,
+  minWidth: 'auto',
```

Comment:
If no `minWidth` is provided we likely shouldn't add extra css tags unless `auto` is desired here. I assume it is added to prevent the `table-styles` code from breaking without a `minWidth`? If so, then we should only apply that style if `minWidth` is specified.

Reply:
removed